reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+
+### Localization and Globalization 
+
+The Kubernetes project community is distributed around the world, and there the
+end user community is the same. Over the past year, the number of international
+contributors has grown, as have initiatives to support localizations of the
+project. 
+
+SIG Docs hosts all localizations of the documentation, but each localization
+has its own group of maintainers and leads. To manage the growing number of
+localizations, SIG Docs started the Localization subproject. Aditionally, SIG
+UI added support for several new localizations of the Kubernetes dashboard. 
+
+### Fostering Inclusivity
+
+The Kubernetes [core values] are critical to the success of project.
+In 2020 we reinforced our focus on inclusivity by [requiring our community 
+leaders]
+to further their education on recognizing unconscious bias and working towards
+creating a more welcoming environment for every contributor. In addition to
+requiring our current leaders to take these steps, it is now a prerequisite for
+any future leads before they take a leadership position.
+
+Our talented moderation teams continue to ensure that all our communication
+channels are safe and inclusive spaces for our contributor base.
+
+During the Black Lives Matter protests in 2020, the project was introspective in
+how it's values intersected with a global movement around equality. We decided
+to make a statement about the importance of inclusivity to where the Kubernetes
+project is today, and how racism doesn't have a place in our project.
+
+[core values]: https://www.kubernetes.dev/community/values/
+[requiring our community leaders]: https://groups.google.com/u/1/g/kubernetes-dev/c/5gRUxPi5XxY/m/1Ollffx4CQAJ
+
+### #shoutouts
+
+A meta accolade.
+
+The #shoutouts channel on the [Kubernetes slack](slack.k8s.io) and highlights
+on [@k8scontributors] in the last year has kept us going. Thank a member of the
+community here and read the past achievements of many. 
+
+[@k8scontributors]: https://twitter.com/K8sContributors
+
+### Remembering Dan Kohn
+
+In [November of 2020, Dan Kohn], the former director of the CNCF sadly passed
+away from complications with Colon Cancer. Dan was instrumental in shaping both
+the Kubernetes project, the CNCF, and the Cloud Native community as a whole. He
+understood that a foundation built on a vibrant and diverse community was a
+requirement to be successful, and the project would not be what it is today
+without him. 
+
+[November of 2020, Dan Kohn]: https://kubernetes.io/blog/2020/11/02/remembering-dan-kohn/
+
+
+## Themes
+
+The following themes emerged from multiple community groups reporting in with 
+similar experiences - whether positive or challenging - and areas of research to
+explore more in the future. 
+
+### Project communication strategy 
+  
+During major growth phases, the Steering Committee needed the various [community
```

Comment:
What's a "major growth phase"?

Reply:
Maybe "During the earlier development phases of the project" ?

Reply:
During major growth phases when contributor growth was doubling year after year, the Steering Committee (or is that too long?)